plugins/language_marte.lua

```diff
@@ -0,0 +1,42 @@
+-- mod-version:3
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "MARTe",
+  files = { "%.mrt$", "%.marte$" },
+  comment = "//",
+  block_comment = { "/*", "*/" },
+  patterns = {
+    { pattern = "//.*",                        type = "comment"  },
+    { pattern = { "/%*", "%*/" },           type = "comment"  },
+    { pattern = { '"', '"', '\\' },            type = "string"   },
+    { pattern = { "'", "'", '\\' },            type = "string"   },
+    { pattern = "%-?%.inf",                    type = "number"   },
+    { pattern = "%.NaN",                       type = "number"   },
+    {
+      pattern = "Class%s+()=()%s+[%a_][%w_:]*",
+      type = { "keyword", "operator", "keyword2"}
+    },
+    {
+      pattern = "Type%s+()=()%s+[%a_][%w_]*",
+      type = { "keyword", "operator", "keyword2"}
+    },
+    {
+      pattern = "[%+%$][%a_][%w_]+%s()=",
+      type = {"function", "operator"}
+    },
+    { pattern = "=%s+()[%a_][%w_]+", type = "string" },
```

Comment:
This is dividing the match in two `()`, but only a single token type is provided.